tools/metrics/export/mongo-export.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mongo-export
+  namespace: {{KUBE_NAMESPACE}}
+spec:
+  template:
+    spec:
+      containers:
+      - name: sling
+        image: slingdata/sling
+        command: ["/bin/sh", "-c"]
+        args:
+          - |
+            # Use the environment variables directly
+            export MONGODB="mongodb://$MONGO_USER:$MONGO_PASS@$MONGO_HOST:$MONGO_PORT/?authSource=database"
+
+            # Run the Sling command
+            sling run --src-conn MONGODB --src-stream 'database.TaskData' --tgt-conn S3 --tgt-object "s3://{{BUCKET_NAME}}/{{FILENAME}}_TaskData.json"
```

Comment:
We only want to export the TaskData collection ?

Reply:
For now.. the imports have to be done per collection so this won't be "generalized" until I decide on how to do imports.